imports/ui/components/organisms/ContentBox.jsx

```diff
@@ -101,10 +102,12 @@ export const ContentBox = ({ content }) => {
    */
 
   let Content
+  const customHTML = content.customHTML || '<div>Hello, Inserted HTML </div>'
 
   switch (type) {
     case 'TEXT':
-      Content = <TextBox {...colors}>{text}</TextBox>
+      Content = <CustomBoxHTML customHTML={customHTML} {...colors} />
+      // Content = <TextBox {...colors}>{text}</TextBox>
```

Comment:
Text should stay as Content. The customHTML will be shown always and independent of the content.type. So it is outside of the switch statement.